keyutil/key_test.go

```diff
@@ -540,3 +540,41 @@ func TestVerifyPair(t *testing.T) {
 		})
 	}
 }
+
+func TestInsecure(t *testing.T) {
+	tests := []struct {
+		name    string
+		run     func(t *testing.T) error
+		wantErr bool
+	}{
+		{"ok RSA 2048", func(t *testing.T) (err error) {
```

Comment:
I'd include a call to `t.Helper` in the runners of this table-driven test.

Reply:
Because I'm not doing any `t.Error'-like method, there's no point in using `t.Helper()` to skip the function in the error information. I'm just passing `testing.T` to use the `Cleanup` method, I can totally avoid it with a defer too.